src/main/database/dummy_data/jobs.ts

```diff
@@ -1,90 +1,115 @@
+import { v4 as uuidv4 } from 'uuid';
+
 const jobs = [
   {
-    uid: 'job1',
-    modelUid: 'model1',
-    startTime: new Date('2023-10-01T10:00:00Z'),
-    endTime: new Date('2023-10-01T12:00:00Z'),
+    uid: uuidv4(),
+    modelUid: 'isr-model',
+    startTime: new Date('2023-10-01T10:14:00Z'),
+    endTime: new Date('2023-10-01T12:09:00Z'),
     status: 'Completed',
     inputs: [
       {
-        path: '/input/path1',
-        path_type: 'temp',
+        path: 'C:/Users/LENOVO/Pictures/Input',
+        path_type: 'local',
       },
     ],
     outputs: [
       {
-        path: '/output/path1',
-        path_type: 'temp',
+        path: 'C:/Users/LENOVO/Pictures/Output',
+        path_type: 'local',
       },
     ],
     parameters: [
       {
-        param1: 'value1',
-      },
-      {
-        param2: 'value2',
+        scale: 4,
       },
     ],
     response: {
-      message: 'Job completed successfully',
+      message: 'Job completed successfully.',
       code: 200,
     },
   },
   {
-    uid: 'job2',
-    modelUid: 'model2',
-    startTime: new Date('2023-10-02T14:00:00Z'),
-    endTime: new Date('2023-10-02T16:00:00Z'),
+    uid: uuidv4(),
+    modelUid: 'sbf-model',
+    startTime: new Date('2023-10-02T14:19:08Z'),
+    endTime: new Date('2023-10-02T14:19:49Z'),
     status: 'Failed',
     inputs: [
       {
-        path: '/input/path2',
-        path_type: 'temp',
+        path: 'F:/USB/Forensic/Input',
+        path_type: 'USB Drive',
```

Comment:
The `path_type` is actually a misnomer. It's a custom key that will be used by the model devs when running inference from the frontend. For example, for SBF, the inputs array could look like:

```
[
  {
    path: "/path/to/input/data",
    path_type: 'Known Dataset'
  },
  {
    path: "/path/to/new/input",
    path_type: 'Target Dataset'
  },
]
```

Reply:
Ah, that makes more sense. Will fix.

Reply:
I have changed it to `path_key` instead of `path_type`.